src/universal/components/Row/RowInfoCopy.ts

```diff
@@ -0,0 +1,8 @@
+import styled from 'react-emotion'
+import {ROW_SUBHEADING_STYLES} from 'universal/styles/rows'
+
+const RowInfoCopy = styled('div')({
+  ...ROW_SUBHEADING_STYLES
```

Comment:
lets keep those styles inside this component & then if another component needs to based off of this one we can use `RowInfoCopy.withComponent`